src/utils/calc.rs

```diff
@@ -275,6 +262,46 @@ impl SlotCalculator {
     }
 }
 
+impl FromEnv for SlotCalculator {
+    type Error = FromEnvErr<ParseIntError>;
+
+    fn inventory() -> Vec<&'static EnvItemInfo> {
+        vec![
```

Comment:
these are now optional, yes? and the inventory should include `CHAIN_NAME`, along with a description of behavior